cmd/derper/derper.go

```diff
@@ -210,22 +212,39 @@ func main() {
 	mux.HandleFunc("/bootstrap-dns", tsweb.BrowserHeaderHandlerFunc(handleBootstrapDNS))
 	mux.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		tsweb.AddBrowserHeaders(w)
+		abuseMailto := ""
+		tailscalePolicies := ""
+		if *showTailscaleSpecificAbuseInformation {
```

Comment:
The reason for this is that when parties other than Tailscale are running DERP servers, we presumably don't want to direct people to Tailscale's policies and security department.